src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -297,6 +297,8 @@ g.test('it_is_invalid_to_have_an_output_attachment_texture_with_non_renderable_f
 
     const descriptor = t.getDescriptor({ width: 1, height: 1, format });
 
+    await t.selectDeviceOrSkipTestCase(info.extension);
```

Comment:
nit: Sorry, other nit was supposed to apply to all instances. Please move this earlier here too.

Reply:
Done.